packages/plugin-search/test/integration/spec/search.js

```diff
@@ -0,0 +1,511 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '../..';
+
+import CiscoSpark from '@ciscospark/spark-core';
+import {assert} from '@ciscospark/test-helper-chai';
+import testUsers from '@ciscospark/test-helper-test-users';
+import retry from '@ciscospark/test-helper-retry';
+
+import '@ciscospark/plugin-conversation';
+import '@ciscospark/plugin-wdm';
+
+import {map, countBy} from 'lodash';
+import uuid from 'uuid';
+import util from 'util';
+
+describe(`plugin-search`, () => {
+  describe(`#people`, () => {
+    let bot, checkov, mccoy, spark;
+
+    before(() => testUsers.create({count: 2})
+      .then((users) => {
+        [checkov, mccoy] = users;
+
+        spark = new CiscoSpark({
+          credentials: {
+            authorization: checkov.token
+          }
+        });
+
+        return spark.mercury.connect();
+      })
+      .then(() => spark.request({
+        api: `hydra`,
+        resource: `bots`,
+        method: `POST`,
+        body: {
+          displayName: `Personal Assistant`,
+          email: `${uuid.v4()}@sparkbot.io`
+        }
+      }))
+      .then((results) => {
+        bot = results.body;
+      })
+    );
+
+    after(() => {
+      spark.mercury.disconnect();
+    });
+
+    it(`searches for users with a specific string`, () => {
+      return spark.search.people({query: `abc`})
```

Comment:
for most (all?) of these tests, you can remove the braces and the return statement. e.g.

```javascript
it(`searches for users with a specific string`, () => spark.search.people({query: `abc`})
  .then((user) => assert.isArray(users)));
```